jack-and-the-beans/buddies-ios
# Edit Profile

Also fixes #64 - Editing a photo actually crops it now!

To test
- Account setup still works
- You can now tap "Edit" at the top right of your profile
- All fields should work
    - Make sure you see things change on a second device.

Thoughts
- Might conflict with @noahtallen's changes. I'm now using `imageVersion` which gets used in the local image cache filename.
- We also want to merge this backend change: https://github.com/jack-and-the-beans/buddies-ios/pull/122#issue-266315952
